org.eclipse.forkexample.calc/src/org/eclipse/example/calc/internal/Calculator.java

```diff
@@ -35,7 +35,9 @@ public Calculator(TextProvider textProvider) {
 		setupDefaultOperations();
 	}
 
+	/* Hier werden die Operationen angeboten */ 
 	private void setupDefaultOperations() {
+		/*more operations to come*/
```

Comment:
Na jetzt wird hier in Deutsch und Englisch kommentiert? Das gefällt mir nicht. ;-)